### DISCUSSION TITLE
How do you handle a field with a name of 'from'

### DISCUSSION THREAD

**Discussion by boatcoder** [encode/django-rest-framework#8058] [Potential Issue] [answered]
Obvlously I can't use

    from = serializers......

Because that is invalid Python

I've written the whole question up here: https://stackoverflow.com/questions/68120966/how-can-you-deal-with-a-reserved-word-from-in-a-serializer-in-drf but don't have a better answer than my smelly method of solving this problem.

**Comment by lovelydinosaur** [comment, marked as answer]
There isn't any graceful way around this, nope.

**Reply by boatcoder** [reply]
I was  afraid you would say that :-(
Looks like there is a need for `source=` and `dest=` but it seems like a small usecase.....
